### PR TITLE
Drag and Drop to the table view fix

### PR DESCRIPTION
**Pull request by Flight**
Drag and Drop to the table view fix

### CODE REVIEW

File: filer/private/sass/layout/_main.scss [hunk 1]
```diff
@@ -689,7 +689,6 @@ body {
 }
 .dropzone-upload-info {
     margin-top: 10px;
-    // make sure that file name on upload progress is cut #675
```
Comment (lory87) [inline filer/private/sass/layout/_main.scss]: why do you remove my comment?

Comment (vxsx) [inline filer/private/sass/layout/_main.scss]: would keep the comment

Comment (Flight) [inline filer/private/sass/layout/_main.scss]: do we need it here?

Comment (lory87) [inline filer/private/sass/layout/_main.scss]: actually on every change. Can you add some comments to your changes as well with pull request id?

File: filer/static/filer/js/addons/table-dropzone.js [hunk 2]
```diff
@@ -47,14 +47,21 @@
...
+                            infoMessage.removeClass(hiddenClass);
+                        }
                         dropzone.addClass(dragHoverClass);
 
-                        folderName.text($(dragEvent.target).parents(dropzoneSelector).data('folder-name'));
+                        folderName.text(folderTitle);
                     },
                     dragleave: function (dragEvent) {
                         var target = $(dragEvent.target);
+                        var folderTitle = target.parents(dropzoneSelector).data('folder-name');
```
Comment (vxsx) [inline filer/static/filer/js/addons/table-dropzone.js]: i know it's unlikely to have nested dropzones, but would still prefer `closest`

**Comment by vxsx** [comment]
overall :+1:

**Comment by lory87** [comment]
:+1:

**Pull request merged**